test/list_path_observer_test.dart

```diff
@@ -1,10 +1,10 @@
-// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
+// Copyright (c) 2013, the Dart project authors.  Please see the AUTHOR file
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
-
+@Skip("Until ported")
```

Comment:
Just curious, what needs to be "ported" for the tests to work? Are there other breaking changes besides the Symbol->String change?

if so would be good to put in https://github.com/dart-lang/observe/blob/master/CHANGELOG.md